lib/usage.js

```diff
@@ -128,12 +128,14 @@ module.exports = function (yargs, y18n) {
   self.help = function () {
     normalizeAliases()
 
-    var demanded = yargs.getDemanded()
+    var demandedOption = yargs.getDemandedOption()
+    var demandedCommand = yargs.getDemandedCommand()
```

Comment:
Yea, so, I kept typing the plural version all the time. I feel like you're right, and we should probably keep this at plural; seems to make more lexical sense.